src/sdk/modules/passkeyValidator/toPasskeyValidator.ts

```diff
@@ -0,0 +1,248 @@
+import { startAuthentication } from "@simplewebauthn/browser"
```

Comment:
This isn't in your dependencies 🤔 . Users won't have this when they install our package.

Reply:
Won't this cause issues for cjs builds? 🤔

Reply:
I think bun will install devDeps when you run `bun i` so everything appears fine when it isn't

Reply:
Yea, I was working with local linked package and didn't notice this, thanks.